conans/test/assets/pkg_cmake.py

```diff
@@ -0,0 +1,49 @@
+import textwrap
+
+from conans.model.ref import ConanFileReference
+from conans.test.assets.cmake import gen_cmakelists
+from conans.test.assets.sources import gen_function_h, gen_function_cpp
+
+
+def pkg_cmake(name, version, requires=None):
+    refs = [ConanFileReference.loads(r) for r in requires or []]
+
+    conanfile = textwrap.dedent("""\
+        from conans import ConanFile
+        from conan.tools.cmake import CMake
+        class Pkg(ConanFile):
+            name = "{name}"
+            version = "{version}"
+            exports = "*"
+            {deps}
+            settings = "os", "compiler", "arch", "build_type"
+            generators = "CMakeToolchain", "CMakeDeps"
+
+            def build(self):
+                cmake = CMake(self)
+                cmake.configure(source_folder="src")
+                cmake.build()
+
+            def package(self):
```

Comment:
we might consider in the future replace this with the autopackage.

Reply:
Yes, agree, it felt a bit early at the moment.